### ISSUE TITLE
Bug: Investigate remaining accessibility issues in analysis portfolio site

### ISSUE THREAD

**Issue by jparr** [cal-itp/data-analyses#1802]
**Where did the bug occur?**
Select from the below, and be sure to affix the appropriate label to this issue (e.g. `dataset`, `jupyterhub`, `metabase`, `analysis.calitp.org`)
- [ ] Data (the warehouse)
- [ ] JupyterHub
- [ ] Metabase
- [ x] analysis.calitp.org  
- [ ] Other (add detail)

**Describe the bug**
Investigate accessibility issues highlighted here: https://github.com/cal-itp/data-analyses/issues/1604#issuecomment-3573367479

**Comment by raebot** [comment]
We've upgraded to JupyterBook v2 and, like with JB v1, there are some accessibility warnings that come from the book theme template itself. For instance, accessibility checks for each portfolio site will warn that the site header logo does not have alt text. There is no way to add alt text via the myst.yml config. I created a GH issue for the maintainers of the myst themes to consider adding the option to add alt text https://github.com/jupyter-book/myst-theme/issues/804#issue-3929583517.

Update: the maintainers responded quickly and have already enabled adding alt text!

**Comment by raebot** [comment]
@edasmalchi  It looks like the JupyterBook v2 upgrade cleared up many of the specific accessibility errors we were seeing with v1 that were more related to the template than any content rendered from a notebook. There are still many issues related to content rendered from notebooks. @erikamov and I were able to resolve some of those (see commits). It sounds like the intention is for analysts to work through many of those. Is that right? Some of the warnings come from packages we're using to render things like maps (folium/branca - see https://github.com/cal-itp/data-analyses/issues/1802#issuecomment-3899124141). So, those aren't in our direct control, but can open issues and maybe even contribute to those project to resolve them. 

Another recommendation I have for reviewing these warnings is to download the axe DevTools extension for Chrome and run the free full page scan. That will give you an interactive way of browsing through the issues. Thanks @erikamov for the reminder to use this!
